bower.json

```diff
@@ -33,7 +33,7 @@
     "angular-ui-router": "~0.2.15",
     "font-awesome": "~4.4.0",
     "jquery": "~2.1.0",
-    "lodash": "~3.9.0",
+    "lodash": "~4.12.0",
```

Comment:
We'll need to check all the lodash calls, there may be some incompatibilities between the major versions..